renku/api/repository.py

```diff
@@ -353,10 +354,15 @@ def init_external_storage(self, force=False):
             cwd=str(self.path.absolute()),
         )
 
-    def track_paths_in_storage(self, *paths):
+    def lfs_installed(self):
+        """Check that Large File Storage is installed."""
+        return HAS_LFS and self.git.config_reader(
+            config_level='repository'
+        ).has_section('filter "lfs"')
+
+    def track_paths_in_storage(self, *paths, lfs=True):
```

Comment:
```suggestion
    def track_paths_in_storage(self, *paths, use_external_storage=True):
```

Reply:
shouldn't we go the other way (call it `lfs` everywhere instead of `external_storage`?

Reply:
We had the discussion about using technology agnostic names in the past ...

Reply:
Indeed, I remember it well, but at some point I think we might just be causing confusion atm. I'm open to being convinced otherwise though.

Reply:
We can provide multiple storage options in the future.